File: nomad-samples/admin/py/basic-live-channel-1.1/schedule_event/remove_asset_schedule_event.py
from constants.project_constants import *
from exceptions.api_exception_handler import *

import json, requests

def remove_asset_schedule_event(AUTH_TOKEN, CHANNEL_ID, SCHEDULE_EVENT_ID):
    API_URL = f"{ADMIN_URL}/liveChannel/{CHANNEL_ID}/liveScheduleEvent/{SCHEDULE_EVENT_ID}"

    # Create header for the request
    HEADERS = {
        'Content-Type': 'application/json',
        "Authorization": "Bearer " + AUTH_TOKEN
    }

    try:
        # Send the request
        RESPONSE = requests.delete(API_URL, headers= HEADERS)
    
        
        if not RESPONSE.ok:
            raise Exception()
        
        return json.loads(RESPONSE.text)
    
    except:
        api_exception_handler(RESPONSE, "Remove Asset Schedule Event failed")

File: nomad-samples/admin/js/basic-live-channel/schedule-event/remove-asset-schedule-event.js
import * as prjConstants from "../constants/project-constants.js";
import apiExceptionHandler from "../exceptions/api-exception-handler.js";

export default async function removeAssetScheduleEvent(authToken, CHANNEL_ID, SCHEDULE_EVENT_ID) 
{
    // Create header for the request
    const HEADERS = new Headers();
    HEADERS.append("Content-Type", "application/json");
    HEADERS.append("Authorization", `Bearer ${authToken}`);

    // Send the request
    const RESPONSE = await fetch(`${prjConstants.ADMIN_API_URL}/LiveChannel/${CHANNEL_ID}/liveScheduleEvent/${SCHEDULE_EVENT_ID}`, {
        method: "DELETE",
        headers: HEADERS
    });

    if (RESPONSE.ok) {
        return;
    }

    await apiExceptionHandler(RESPONSE, "Remove Asset Schedule Event failed");
}